agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -561,14 +562,16 @@ bool backhaul_manager::finalize_slaves_connect_state(bool fConnected,
             return false;
         }
 
+        auto db = AgentDB::get();
+
         std::string strIface;
         network_utils::iface_info iface_info;
         bool backhaul_manager_exist = false;
 
         notification->params().is_prplmesh_controller = is_prplmesh_controller;
         notification->params().controller_bridge_mac = tlvf::mac_from_string(controller_bridge_mac);
 
-        if (!local_gw) {
+        if (!db->device_conf.local_gw) {
```

Comment:
```suggestion
        if (!local_gw) {
```
if above code change suggestion is adopted then this should remain the same
or replace with the alternative name you choose

(relevant to all occurrences)